file-formats/chdo/type/zif_aff_chdo_v1.intf.abap

```diff
@@ -0,0 +1,98 @@
+INTERFACE zif_aff_chdo_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! <p class="shorttext">Table or Structure</p>
+    "! Table or Structure
+    "! $showAlways
+    BEGIN OF ty_tablestruc,
+      "! <p class="shorttext">Name of Table or Structure</p>
+      "! Name of table or structure
+      "! $showAlways
+      name                   TYPE c LENGTH 30,
+      "! <p class="shorttext">Log Multiple Changes</p>
+      "! Specifies whether multiple changes can be logged
+      "! $showAlways
+      multiple_changes        TYPE abap_bool,
+      "! <p class="shorttext">Log Field Values for Deletions</p>
+      "! Logging of field values for deletions.
+      "! Logging of initial values depends on setting 'Log Initial Values for Deletions'
```

Comment:
Guidance for this kind of texts:
Either repeat the title text above (sentence case) or provide a help text.

Reply:
```suggestion
      "! Log field values for deletions.
      "! Logging of initial values depends on setting 'Log Initial Values for Deletions'
```